pkgs/development/compilers/sapmachine/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchurl, unzip }:
+
+let
+  version = "13.0.1";
+
+  sha256_linux = "21b2a4d3d80bb8434235e9416d347967ca4714e1c54e49136e739b8447c87e56";
+  sha256_darwin = "f0111c1cba6d1b1042724df7d28f2a274981bd90fbfe6c940ca5cc9ceddd8825";
+
+  platform = if stdenv.isDarwin then "osx" else "linux";
```

Comment:
you can move all this attributes inside the derivation

Reply:
moved inside ... thank you